engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/HistoricIncident.xml

```diff
@@ -388,23 +388,29 @@
     <result property="removalTime" column="REMOVAL_TIME_" jdbcType="TIMESTAMP"/>
   </resultMap>
 
+  <resultMap id="historicIncidentResultWithRootCauseIncidentMessageMap" type="org.camunda.bpm.engine.impl.persistence.entity.HistoricIncidentEntity" extends="historicIncidentResultMap">
+    <result property="rootCauseIncidentMessage" column="ROOT_CAUSE_INCIDENT_MESSAGE_" jdbcType="VARCHAR"/>
```

Comment:
Nitpick:
to keep consistent and short lets also abbreviate `message` here like in the other message field:
```suggestion
    <result property="rootCauseIncidentMessage" column="ROOT_CAUSE_INCIDENT_MSG_" jdbcType="VARCHAR"/>
```